src/index.html

```diff
@@ -10,13 +10,98 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="menu">
+        <ul>
+          <li>
+            <a
+              class="nav_link is-active"
+              href="#apple"
+            >
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `nav_link` should be `nav__link` to follow the BEM naming convention.